### PR TITLE
Fix example usage

### PR DESCRIPTION
**Pull request by elyase**

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -67,26 +67,26 @@ Example Usage
 
 ::
 
+    from locasticsearch import Locasticsearch
     from datetime import datetime
-    from elasticsearch import Elasticsearch
-    es = Elasticsearch()
+
+    es = Locasticsearch()
```
Comment (sethmlarson) [inline docs/index.rst:73]: What is locasticsearch? Typo?

File: docs/index.rst [hunk 2]
```diff
@@ -67,26 +67,26 @@ Example Usage
...
+        "text": "Elasticsearch: cool. bonsai cool.",
+        "timestamp": datetime(2010, 10, 10, 10, 10, 10),
     }
-    res = es.index(index="test-index", id=1, body=doc)
-    print(res['result'])
+    res = es.index(index="test-index", doc_type="tweet", id=1, body=doc)
 
-    res = es.get(index="test-index", id=1)
-    print(res['_source'])
+    res = es.get(index="test-index", doc_type="tweet", id=1)
```
Comment (sethmlarson) [inline docs/index.rst:82]: With 7.x we shouldn't use `doc_types` in examples, why are we using this here?

File: docs/index.rst (hunk 1 above)
Comment (elyase) [reply docs/index.rst]: it is a new POC library I am experimenting with. But definitely out of place here :-)

File: docs/index.rst (hunk 2 above)
Comment (elyase) [reply docs/index.rst]: I was also confused. Stripped them away now

**Comment by sethmlarson** [comment]
@elyase I'm not seeing what is improved with this diff, can you explain the rationale?

**Comment by elyase** [comment]
Well this is somewhat embarrassing but I just realized that I was trying to fix something that is already fixed. To make things worse I just noticed I myself submitted the [same pull request](https://github.com/elastic/elasticsearch-py/pull/923) last year 🤦‍♂️.

I think I must have opened an old version of the documentation, is the only explanation I can think of. So sorry for the waste of time and lets pray it doesn't happen again next year :-)

**Comment by sethmlarson** [comment]
@elyase No problem! You just wanted to make sure that issue was **definitely fixed** :wink: Thanks for your contributions.

**Pull request closed without merging**